cli/index.ts

```diff
@@ -184,7 +185,21 @@ async function serve(port: number, pathToSpec: string, options: Options = {}) {
           log('Updated successfully');
         } catch (e) {
           console.error('Error while updating: ', e.message);
-        }})
+        }
+      })
+      .on('add', async path => {
+        log(`File ${path} added, updating docs`);
+        try {
```

Comment:
These five lines are exactly the same as above in `change` handler. I think there is a point to create a function